examples/TSSOP-20.example.tsx

```diff
@@ -1,12 +1,13 @@
 import { JsCadFixture } from "jscad-fiber"
 import { ExtrudedPads } from "../lib/ExtrudedPads"
-import { Tssop } from "lib/Tssop"
+import { Footprinter3d } from "lib/Footprinter3d"
```

Comment:
There is a directory of examples that should use footprinter3d, they’re all named specifically with the footprinter string